### DISCUSSION TITLE
InMemoryConversation

### DISCUSSION THREAD

**Discussion by dbconrad1** [llm-tools/embedJs#93] [Q&A]
According to the documentation I should be able to reference "import { InMemoryConversation } from '@llm-tools/embedjs/conversation/memory';" to utilize the InMemoryConversation functionality but I get "Cannot find module" error.  I'm able to reference other areas of the module so I know it's loaded properly and I see the relevant files in the node_modules directory.  Trying other obvious paths results in the same.  What am I missing?

**Comment by adhityan** [comment]
Weird, this should be working. Let me take a look.

**Comment by adhityan** [comment]
The library, it's imports and documentation are significantly updated since the move to monorepo. This was recently tested and working.
